lib/ViewModels/FeatureInfoPanelViewModel.js

```diff
@@ -90,11 +112,39 @@ FeatureInfoPanelViewModel.prototype.closeIfClickOnBackground = function(viewMode
     return true;
 };
 
+FeatureInfoPanelViewModel.prototype.close = function() {
+    this.isVisible = false;
+    // this.resetSections();  // would briefly flash an empty feature info panel as it closes
```

Comment:
Right, because the panel fades out when isVisible is set to false, rather than disappearing instantaneously.  If resetting the sections is valuable, you could arrange to do it later (though it's a bit ugly).  If it's not needed, might as well just remove this commented-out code.